### PR TITLE
Motd scripts

### PR DESCRIPTION
**Pull request by javiersantossanchez**
Initial version of file to be execute on login into console.

Add validation to show information when login

add validation to show system information once y day 
using the motd framework  
Add validation using .hushlogin file

Add feature to disable the message when the .hushlogin file
is added to home folder
Improve the format on messages

show information about wsl version 

Change permission on the file  

copy files into /etc/update-motd.d folder

copy the scripts to show messages on login 
Improving messages

Update Text

Add permission of execution to motd scripts

Fixing path of files

### CODE REVIEW

File: update-motd.d/30-pengwin-setup [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+printf " Pengwin include 'Pengwin Setup', allows new Pengwin users to get started on Linux immediately"
```
Comment (crramirez) [inline update-motd.d/30-pengwin-setup:3]: Pengwin includes 'Pengwin Setup,' which allows new Pengwin users to get started on Linux immediately

File: update-motd.d/30-pengwin-setup [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+printf "\n"
+printf " To execute run: pengwin-setup"
```
Comment (crramirez) [inline update-motd.d/30-pengwin-setup:5]: To execute, run: pengwin-setup

File: update-motd.d/30-pengwin-setup [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+printf "\n"
+printf " To check for updates run: pengwin-setup update"
```
Comment (crramirez) [inline update-motd.d/30-pengwin-setup:7]: To check for updates, run: pengwin-setup update

File: update-motd.d/40-environment [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+printf " Environment variables:\n"
+printf "  DISPLAY    %s\n" "$DISPLAY"
+printf " \n"
+printf " Network\n"
+ip -o -f inet addr show | awk '{printf "  IPv4 for %-20s %s\n",$2, $4}'  |cut -f1 -d/
+printf " \n"
+printf " Windows Home Location: "
+printf " %s" "$winhome_path"
+printf " \n"
+printf " The windows drives are mounted at:\n"
```
Comment (crramirez) [inline update-motd.d/40-environment:14]: The Windows drives are mounted at

File: update-motd.d/40-environment [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+wslpath -u 'C:\' | awk '{printf "  %s", $1}'
+printf " \n"
+printf " To open the current folder with windows explorer run: wslview ."
```
Comment (crramirez) [inline update-motd.d/40-environment:17]: To open the current folder with Windows Explorer, run

**Review by crramirez — CHANGES_REQUESTED** [review]

File: update-motd.d/30-pengwin-setup (hunk 1 above)
Comment (javiersantossanchez) [reply update-motd.d/30-pengwin-setup]: Done

File: update-motd.d/30-pengwin-setup (hunk 3 above)
Comment (javiersantossanchez) [reply update-motd.d/30-pengwin-setup]: Done

File: update-motd.d/40-environment (hunk 4 above)
Comment (javiersantossanchez) [reply update-motd.d/40-environment]: Done

File: update-motd.d/40-environment (hunk 5 above)
Comment (javiersantossanchez) [reply update-motd.d/40-environment]: Done

**Pull request merged**